doc/digraph.xml

```diff
@@ -252,6 +252,89 @@ gap> DigraphVertexLabel(gr, 2);
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="DigraphEdgeLabels">
+<ManSection>
+  <Oper Name="DigraphEdgeLabels" Arg="digraph"/>
+  <Oper Name="SetDigraphEdgeLabels" Arg="digraph, labels"/>
+  <Oper Name="SetDigraphEdgeLabels" Arg="digraph, func"/>
+  <Description>
+    If <A>digraph</A> is a digraph, then <C>DigraphEdgeLabels</C> returns a
+    copy of the labels of the edges in <A>digraph</A> as a list of lists
+    <C>labels</C> such that <C>labels[i][j]</C> is the label on the edge
+    from vertex <A>i</A> to vertex <A>j</A>.
```

Comment:
It seems to me that this is going to lead to problems. Is it maybe desirable to actually use an *edge* (i.e. a pair (i,j)) here, or at least provide the option to do so?